Package.swift

```diff
@@ -1,10 +1,11 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.0
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "ETDataDrivenViewKit",
+    platforms: [.iOS(.v9), .macOS(.v10_10), .watchOS(.v2), .tvOS(.v9)],
```

Comment:
Myslim, ze treba na takovejch hodinkach nam tahle knihovna bude uplne k nicemu. Ale zatim to tam nechame... nicemu by to nemelo vadit.